src/index.ts

```diff
@@ -1,17 +1,17 @@
 /**
  * Note: Entry point for _browser_ build is in browser/index.ts
  */
-import EventSource from 'eventsource'
+import EventSourcePolyfill from 'eventsource'
 import {groqStore as groqStoreApi} from './groqStore'
-import {Config, EnvImplementations, GroqStore} from './types'
+import {Config, GroqStore} from './types'
 import {getDocuments} from './node/getDocuments'
 import {assertEnvSupport} from './node/support'
 
 export function groqStore(config: Config): GroqStore {
   assertEnvSupport()
 
   return groqStoreApi(config, {
-    EventSource: (EventSource as any) as EnvImplementations['EventSource'],
+    EventSource: config.EventSource ?? EventSourcePolyfill,
```

Comment:
I just now realize that we where already depending on eventsource. Ideally this would be provided from the outside or at least dynamically imported; but lets not touch it in this feature change. My whole point of of saving on bundlesize sorta goes out the window here.

Since next-sanity loads groq-store async its not too bad I guess.

Reply:
Good point! Well this way it is more configurable, so one can bring a wacky event source if desired